clusterloader2/pkg/test/simple_test_executor.go

```diff
@@ -60,13 +60,20 @@ func (ste *simpleTestExecutor) ExecuteTest(ctx Context, conf *api.Config) *error
 	if err := ctx.GetChaosMonkey().Init(conf.ChaosMonkey, stopCh); err != nil {
 		return errors.NewErrorList(fmt.Errorf("error while creating chaos monkey: %v", err))
 	}
-	automanagedNamespacesList, err := ctx.GetClusterFramework().ListAutomanagedNamespaces()
+	automanagedNamespacesList, staleAutomanagedNamespacesList, err := ctx.GetClusterFramework().ListAutomanagedNamespaces()
```

Comment:
Drop `List` suffix. 

I know the convention in this file is different, but this naming seems to be against Go convention of using short and concise names. Type of a variable can be easily inferred from its type ;)